### DISCUSSION TITLE
ligo-segments-1.4.0-py310 doesn't show up in conda-forge linux-64 repodata.json

### DISCUSSION THREAD

**Discussion by duncanmmacleod** [conda/infra#11] [General]
The conda-forge linux-64 repodata doesn't include `ligo-segments-1.4.0-py310*` which is present on anaconda.org:

```console
$ curl -sL https://conda.anaconda.org/conda-forge/linux-64/repodata.json | grep ligo-segments-1.4.0-py310
<empty>
```

The same issue does not appear for `win-64` or `osx-64`:

```console
$ curl -sL https://conda.anaconda.org/conda-forge/osx-64/repodata.json | grep ligo-segments-1.4.0-py310
    "ligo-segments-1.4.0-py310he24745e_2.tar.bz2": {
```

I can't see anything in the repodata-patches feedstock that would impact this. Any help is greatly appreciated.

EDIT: the problem can be seem manifest in [this build failure](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=420114&view=logs&jobId=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642) for a downstream project's Python 3.10 migration.

**Comment by duncanmmacleod** [comment]
Is anyone able to help me out with this?

**Comment by beckermr** [comment]
@chenghlee @jezdez?

**Comment by chenghlee** [comment]
Can you take another look? I removed the broken artifacts from the cache, and the sync should have picked up the correct ones.

**Reply by duncanmmacleod** [reply]
@chenghlee, thank you! I can see the packages now. Do you understand what happened, and is there anything I could have done to prevent this, or resolve this issue myself?

**Reply by chenghlee** [reply]
This was a fault on the Anaconda side; during the CDN sync process, some of the package artifacts were corrupted, and our automation was unable to properly recover. I've opened an internal ticket to streamline our alerting and ability to recover.
